hoot-core/src/main/cpp/hoot/core/cmd/ApplyChangesetCmd.cpp

```diff
@@ -98,8 +98,7 @@ class ApplyChangesetCmd : public BaseCommand
       osm.setUrl(args[args.size() - 1]);
 
       //  Create a URL without user info for logging
-      QUrl printableUrl = osm;
-      printableUrl.setUserInfo("");
+      QString printableUrl = osm.toString(QUrl::RemoveUserInfo);
```

Comment:
I added this change in to everywhere that a `QUrl` is used that is printed to the log to remove username and password information.  There are maybe a handful of these changes.